node-operators/networks/collators/requirements.md

```diff
@@ -17,9 +17,10 @@ This guide will help you to get started fulfilling the collator requirements so
 
 ## Community Guidelines {: #community-guidelines }
 
-One of the primary goals of the Moonbeam Foundation is to maintain the decentralization of the Moonriver and Moonbeam networks. To further this goal, the Moonbeam Foundation reserves the right to take action if any of the following occur:
+One of the primary goals of the Moonbeam Foundation is to maintain the decentralization of the Moonriver and Moonbeam networks. To further this goal, the Moonbeam Foundation reserves the right to take action if any of the following forbidden offenses occur:
 
-- An entity is running more than 4 collators in either network
+- An entity is running more than four collators in either network
+- A collator is running multiple nodes using the same Nimbus key causing equivocation. Equivocation is the action of submitting multiple blocks at the same block height and is strictly forbidden due to the network degradation that it implies. This can be done by a malicious actor (trying to get more blocks included/produced) or by mistake (having a backup node running with the same key). Each node needs to have its own unique keys and any backup solutions need to ensure there can be no possibility of equivocation
```

Comment:
Equivocation could easily have its own article(s), so I'm not sure that we want more detail than this.

But there are definitely security implications. Being able to trivially create your own forks allows opportunities for deception.

We should be safe from long-range attacks because of finalization (assuming our relay chain is honest), but without any equivocation defense we still have any sort of nothing-at-stake problem, which is (IMO) more important than performance concerns.

Reply:
Yeah I think, being too descriptive might just cause confusion. I agree with what/how it is currently described